docs/integrations/sources/hubspot.md

```diff
@@ -337,6 +337,7 @@ The connector is restricted by normal HubSpot [rate limitations](https://legacyd
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                                                                          |
 |:--------|:-----------| :------------------------------------------------------- |:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 4.2.13 | 2024-07-18 | [41639](https://github.com/airbytehq/airbyte/pull/41639) | Handle custom object name collisions with standard objects |
```

Comment:
It is missing the metadata.yaml and pyproject connector version update.  Please fix that.

Reply:
@marcosmarxm updated!